alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php

```diff
@@ -544,7 +575,7 @@ private function getIfarmEmbedCode($bundleContent)
                         <body >
                         	<div id="player_container"></div>
                             	<script type = "text/javascript" > ' . $bundleContent . '</script >
-				<script type = "text/javascript" > window.originalRequestReferrer = "' . $_SERVER['HTTP_REFERER'] . '"</script >
+				<script type = "text/javascript" > window.originalRequestReferrer = "' . @$_SERVER['HTTP_REFERER'] . '"</script >
```

Comment:
instead of suppressing this shouldn't this added only if $_SERVER['HTTP_REFERER'] is set?

Reply:
this is the common approach used over the code

Reply:
That now why i am asking 

What will happen if the player gets something like this:
<script type = "text/javascript" > window.originalRequestReferrer = ""</script >

Reply:
The player handles it, if the response is empty "", it goes to fallback